EIPS/eip-1.md

```diff
@@ -3,7 +3,7 @@ eip: 1
 title: EIP Purpose and Guidelines
 status: Active
 type: Meta
-author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>
+author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, [and others](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md).
```

Comment:
Unfortunately I think this will break jekyll's postprocessing. If you want to keep it, can you check by running a local copy?

Reply:
Doesn't render nicely. But it is not broken. Respectfully requesting to merge as is and improve this rendering later.

![screen shot 2018-05-25 at 1 40 23 pm](https://user-images.githubusercontent.com/382183/40558433-4fabfce2-6021-11e8-97ac-988b447b7bf9.png)

Reply:
I agree that this is out of scope for this PR, but it would be nice to fix this rendering. Here is one option: https://stackoverflow.com/questions/28819413/add-link-to-yaml-in-multimarkdown-metadata.

Reply:
Thank you for the link. P.S. presently your review status on this PR is denied. Is anything else outstanding to earn your approved vote?

Reply:
As a quick temporary fix, perhaps we could change this to `, and others: 
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md

This would at least make it easier to copy and paste the link. I'm not sure how this renders, e.g. if newlines will work.

Reply:
@jamesray1 That change would break it

![screen shot 2018-05-31 at 10 52 50 pm](https://user-images.githubusercontent.com/382183/40818521-6413f6ae-6525-11e8-9fcf-c054b99f8d6d.png)

I have added your concern as a Follow-On Work above. I hope this PR can be merged without the requirement of upgrading the website rendering system.

Reply:
Could you try:
```
author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, and others:
            https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md).
```

As per http://fletcher.github.io/MultiMarkdown-4/metadata.html, I think that should work. Using an  XLST file to actually render it as a link could be done later, but most people will probably not bother to look at the link.

Reply:
Same thing 

![screen shot 2018-05-31 at 11 55 15 pm](https://user-images.githubusercontent.com/382183/40820362-153991c0-652e-11e8-8e76-0d7812e8f4f7.png)

Reply:
Same with changes in punctuation:

![screen shot 2018-05-31 at 11 56 00 pm](https://user-images.githubusercontent.com/382183/40820381-3575d05c-652e-11e8-94a6-b64471c8eab1.png)

Reply:
@jamesray1 Bingo

![screen shot 2018-05-31 at 11 57 34 pm](https://user-images.githubusercontent.com/382183/40820424-6a8bbe78-652e-11e8-8d0e-51686d264365.png)

Fixed in 3dd4b31

Reply:
@jamesray1 Does this resolve all your issues? Will you please mark PR as approved.

Reply:
Great, but the change isn't showing above and that commit hash isn't in the history.

Reply:
Oh yeah the extra colon would have stuffed it up. If you put the whole value in quotes that should work, let me try that locally.

https://stackoverflow.com/questions/8783705/how-could-should-i-state-colon-punctuation-in-a-yaml-file

Reply:
I'm having an [issue with jekyll](http://talk.jekyllrb.com/t/jekyll-serve-usr-lib-ruby-gems-2-5-0-gems-bundler-1-16-2-lib-bundler-spec-set-rbin-block-in-materialize-could-not-find-i18n-0-9-5-in-any-of-the-sources-bundler-gemnotfound/1864), which is unrelated to the change in the author filed, so I can't test this.

Reply:
Sorry about that, this time the push went through. Please repeal.